backend/domain/Room.js

```diff
@@ -144,12 +145,12 @@ class Room {
     }
 
     /**
-     * Generate a random alphanumeric id
-     * @returns {string} a random alphanumeric id
+     * Generate a univerally unique identifier
+     * @returns {string} a univerally unique identifier
```

Comment:
Good description, just a typo of the word universally here, not a big deal but just to let you know! Not urgent